modules/nf-core/trinity/main.nf

```diff
@@ -1,7 +1,6 @@
 process TRINITY {
     tag "$meta.id"
     label 'process_high'
-    label 'process_high_memory'
```

Comment:
Why has this gone here?

Reply:
`nf-core` lint gave a warning saying this label is non-standard.

Reply:
A warning or an error?

Reply:
A warning.

Reply:
This is where it originates from: https://github.com/nf-core/tools/blob/930ece572bf23b68c7a7c5259e918a878ba6499e/nf_core/modules/lint/main_nf.py#L425-L461

Reply:
I think we leave this in, and file an issue, because `process_high_memory` is in the template:
https://github.com/nf-core/tools/blob/930ece572bf23b68c7a7c5259e918a878ba6499e/nf_core/pipeline-template/conf/base.config#L52

Reply:
I agree! I'll create a PR on nf-core/tools.

Reply:
Thank you!

Reply:
https://github.com/nf-core/tools/pull/3087